api/http/management.go

```diff
@@ -503,6 +574,41 @@ func (h ManagementController) ConnectServeWS(
 	}
 }
 
+func sendResizeMessage(m *ws.ProtoMsg,
+	sess *model.Session,
+	controlRecorderBuffered *bufio.Writer) (n int) {
+	var height uint16 = 0
+	switch m.Header.Properties[model.ResizeMessageTermHeightField].(type) {
```

Comment:
This looks very fragile

Reply:
I added:
```go
	if _, ok := m.Header.Properties[model.ResizeMessageTermHeightField]; ok {
		return 0
	}
	if _, ok := m.Header.Properties[model.ResizeMessageTermWidthField]; ok {
		return 0
	}
```
is that better? (it is not my fault that occasionally it comes as int and then as uint :) it depends if there was an increase or decrease in dimensions, I guess)

Reply:
What I meant is that we should use reflection to make it independent of integer data-types altogether.

Reply:
I kind of do not believe in using reflection if it can be avoided. do you have a strong opinion here?

Reply:
If it's not going to be a problem if I modify the UI or mender-connect to use int16 or int32 without breaking deviceconnect I'm ok with it.